CHANGELOG.md

```diff
@@ -3,7 +3,29 @@
 ## Unreleased
 
 ### Breaking changes
-You must make the following changes when you migrate to this release, or your service may break.
+Where applicable, you must make the following changes when you migrate to this release, or your service may break.
+
+#### Check your accordions still display as expected
+
+[See the new design and revised guidance for accordions in the Design System](https://design-system.service.gov.uk/components/accordion/). In particular, review the guidance on when and when not to use accordions.
+
+You should check that your accordions still display as expected if you:
+
+- have applied any custom styles to your accordions
+- use accordions in non-standard ways
+- have section headings that contain unusual formatting (for example, wrappers)
+
+You do not need to make changes to the content that accordions hide or show - this release's changes do not affect it.
+
+These changes were introduced in [pull request #2257: Update accordion design](https://github.com/alphagov/govuk-frontend/pull/2257).
```

Comment:
@hannalaakso I remember you were a bit nervous about linking to the PR and/or suggested it needing some work before linking to it from the Changelog. Just cc'ing you in here for awareness.

Reply:
@vanitabarrett Thanks for the nudge! Yes, we want to make sure that the PR has a comprehensive description of changes or links to other places that do.

I'm going to wait until we've got the backlog summary written up and then reflect what else needs go in the PR description apart from linking to the summary.